opm/common/OpmLog/LogBackend.cpp

```diff
@@ -33,23 +33,49 @@ namespace Opm {
 
     void LogBackend::configureDecoration(std::shared_ptr<MessageFormatterInterface> formatter)
     {
-        formatter_ = formatter;
+        m_formatter = formatter;
+    }
+
+    void LogBackend::configureMessageLimiter(std::shared_ptr<MessageLimiter> limiter)
```

Comment:
As I see it you use this method to *set* - an already configured message limiter, so I think `setMessageLimiter()`would be a better name?